docs/JSON-RPC.md

```diff
@@ -234,12 +234,18 @@ Results:
 
 | Name | Type | Description |
 | --- | --- | --- |
+| result.connections | number | The number of active connections. |
 | result.clients | array | The list of connected clients. |
 | result.clients[*].id | number | The client’s channel id. |
 | result.clients[*].address | string | The client’s address (ip:port). |
 | result.clients[*].name | string | The client’s name. |
 | result.clients[*].jitterBufferSize | number | The client’s jitter buffer size. |
 | result.clients[*].channels | number | The number of audio channels of the client. |
+| result.clients[*].instrumentCode | number | The id of the instrument for this channel. |
+| result.clients[*].instrumentName | number | The text name of the instrument for this channel. |
```

Comment:
`instrumentName | number`?  That sounds strange.  What is it meant to represent?  The position in the enum?  In which case it's redundant.

Reply:
>  we could remove the instrument name

I think this is wrongly documented and should be a string (but I still believe it should be removed.)